icu4c/source/i18n/number_formatimpl.cpp

```diff
@@ -250,10 +250,8 @@ NumberFormatterImpl::macrosToMicroGenerator(const MacroProps& macros, bool safe,
         fUsagePrefsHandler.adoptInsteadAndCheckErrorCode(usagePrefsHandler, status);
         chain = fUsagePrefsHandler.getAlias();
     } else if (isMixedUnit) {
-        MeasureUnitImpl temp;
-        const MeasureUnitImpl &outputUnit = MeasureUnitImpl::forMeasureUnit(macros.unit, temp, status);
         auto unitConversionHandler =
-            new UnitConversionHandler(outputUnit.units[0]->build(status), macros.unit, chain, status);
+            new UnitConversionHandler(macros.unit, chain, status);
```

Comment:
Reformat? I would recommend git-clang-format. Run with --diff to just see what it would do instead of changing anything, and you can specify the master commit with --commit, so that it only formats things that have been changed relative to master:

$ git clang-format --diff --commit master

(One example of a clang-format change I would reject: indentation of commented-out unit tests.)

My workflow would be to be in a client where changes have already been committed, then use "git clang-format", then vscode's diffs handling to cycle through all the diffs, stage the ones I like, and revert the ones that I don't like (e.g. "alt f3" for the next change, and the Source Control sidebar to find the changed files). If I understood correctly, you liked doing that with sublime merge?

Reply:
it need to reformatted, done